AsyncDisplayKit/ASCellNode.mm

```diff
@@ -210,9 +210,9 @@ - (void)cellNodeVisibilityEvent:(ASCellNodeVisibilityEvent)event inScrollView:(U
   // To be overriden by subclasses
 }
 
-- (void)visibilityDidChange:(BOOL)isVisible
+- (void)visibileStateDidChange:(BOOL)isVisible
```

Comment:
The new name is growing on me!  It's probably the right choice.

@lappp9 I think we need to re-add a copy of the original method and keep it working for now.  There are probably 100 definitions of this method in Pinterest, and while I can certainly land a find-and-replace, I don't want to make a tight timing connection to updating the framework there.  Other clients will no doubt be affected as well.

For now let's just add the old method without any special warnings so I can do a normal release of the framework.  As soon as we go to 2.0 Beta we will add a deprecation warning, and perhaps with 2.0 GM we can remove it entirely.